front/app/components/ProjectDescriptionBuilder/Settings/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { useEditor, ROOT_NODE } from '@craftjs/core';
+
+import { CONTENT_BUILDER_DELETE_ELEMENT_EVENT } from 'components/admin/ContentBuilder/constants';
+
+import { MessageDescriptor } from 'utils/cl-intl';
+import eventEmitter from 'utils/eventEmitter';
+
+import Settings from '../../admin/ContentBuilder/Settings/Settings';
+import { SelectedNode } from '../../admin/ContentBuilder/Settings/typings';
+
+const ContentBuilderSettings = () => {
+  const { actions, selectedNode, isEnabled } = useEditor((state, query) => {
+    const currentNodeId: string = query.getEvent('selected').last();
+    let selectedNode: SelectedNode | undefined;
+
+    if (currentNodeId) {
+      selectedNode = {
+        id: currentNodeId,
+        name: state.nodes[currentNodeId].data.name,
+        title: state.nodes[currentNodeId].data.custom?.title as
+          | MessageDescriptor
+          | undefined,
+        // TODO: Fix this the next time the file is edited.
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+        settings: state.nodes[currentNodeId].related?.settings,
```

Comment:
Minor. Might be a good opportunity to check if this can be undefined and if not remove the comment
```suggestion
        settings: state.nodes[currentNodeId].related.settings,
```